packages/oui-angular/package.json

```diff
@@ -10,6 +10,7 @@
     "karma:watch": "cross-env NODE_ENV=test babel-node ./node_modules/karma/bin/karma start ../../build/karma.global.conf.js --watch"
   },
   "dependencies": {
+    "@oui-angular/button": "^1.0.0",
```

Comment:
Dependencies not used anymore. (This file neither)

Reply:
Ok, removed